### DISCUSSION TITLE
Unable to install klipperscreen / Stuck on boot screen

### DISCUSSION THREAD

**Discussion by lin-ycv** [Guilouz/Klipper-Flsun-Speeder-Pad#151] [Q&A] [answered]
On the step of installing 1 instance of the official build, after executing `./KlipperScreen/scripts/KlipperScreen-install.sh` 
after a while i get disconnected, and the pad is unresponsive
```
Setting up libqmi-glib5:armhf (1.30.4-1~ubuntu20.04.1) ...
Setting up dnsmasq-base (2.80-1.1ubuntu1.7) ...
client_loop: send disconnect: Connection reset
```
I then rebooted the printer, but it it's stuck on the boot screen that says booting system.

What can i do to resolve the issue?

**Comment by lin-ycv** [comment]
re-did the whole process, again the pad freezes when running kliiperScreen-install, but this time at a different setup
```
Setting up libmpv-dev:armhf (0.32.0-1ubuntu1) ...
Setting up libhttp-negotiate-perl (6.01-1) ...
Setting up libhttp-cookies-perl (6.08-1) ...
Setting up iptables (1.8.4-3ubuntu2.1) ...
client_loop: send disconnect: Connection reset
```
after the connection reset, i'm unable to ssh
is it still installing?the wiki does say `Note: Installation may take several minutes.`, but at this point it's being an hour...
Any help is appreciated.

**Reply by Wrathlord12** [reply]
exactement le même problème depuis ce matin impossible de reinstaller

**Comment by stephanr111** [comment]
I have same Problem with Flsun V400 Nano V2.1
When i install Klipper Screen, inside it lost connection and all is broken.
I tried now 3 Times. 2 Times with Guilouz Klipper Screen and one time with kiauh Klipper Screen. Same Problems.

**Comment by Maggan0** [comment]
Same Problem here. I tried it several times and it lost WiFi and SSH Access while updating or installing Klipper Screen.

**Comment by lin-ycv** [comment, marked as answer]
There's a work-around that will fix the install for now, but may create problems in the future:
https://github.com/KlipperScreen/KlipperScreen/issues/1215#issuecomment-1870172914

**Comment by stephanr111** [comment]
At removing flsun builds i skipped the step remove KlipperScreen. Then reinstall only klipper, mainsail and moonraker with kiauh. Copyed the Guilouz KlipperScreen Uptade Manager in the moonraker.conf and made a hard recovery. New KlipperScreen was installed and it works. Found no issus for now.
